coa_flask_app/contribution.py

```diff
@@ -0,0 +1,117 @@
+"""
+A module designed to hold the logic related to the contribution page.
+"""
+
+import bisect
+import csv
+from datetime import datetime
+from typing import Dict, List
+
+from coa_flask_app.db_accessor import Accessor
+
+
+def get_tls() -> List[str]:
+    """
+    Get all the TLs for the drop downs.
+
+    Returns:
+        The list of all the TLs.
+    """
+    query = """
+            SELECT
+                DISTINCT team_captain
+            FROM coa.team_info
+            ORDER BY team_captain
+            """
+    with Accessor() as db_handle:
+        db_handle.execute(query)
+        return [tl[0] for tl in db_handle.fetchall()]
+
+
+def get_trash_items() -> Dict[str, List[str]]:
+    """
+    Get all the trash items for the drop downs.
+
+    Returns:
+        The mapping of material to trash items.
+    """
+    query = """
+            SELECT
+                DISTINCT item_id,
+                material,
+                category,
+                item_name
+            FROM coa.item
+            """
+    with Accessor() as db_handle:
+        db_handle.execute(query)
+        items = [(row[1],
+                  f'{row[2]}, {row[3]}[{row[0]}]')
+                 for row in db_handle.fetchall()]
+
+    trash_items: Dict[str, List[str]] = {}
+    for material, item in items:
+        if material in trash_items:
+            bisect.insort(trash_items[material], item)
```

Comment:
Minor: Might want to add a comment that this returns the items sorted in alphabetical order.